cli/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "cli"
+version = "0.1.0"
+edition = "2021"
```

We've switched to 2024 everywhere else in the repo

```suggestion
edition = "2024"
```